### ISSUE TITLE
Export & Import Selection

### ISSUE THREAD

**Issue by sy-b** [qarmin/czkawka#802]
TLDR: Add a feature to export & import a list of selected items. So, we **humans** can pause/hibernate/put off & resume the work at will.

---

After a lot of putting off, I finally had the time & courage to take up the mammoth venture of "Cleaning [& reorganizing] the junk." I decided to start with the easiest part  — Images. Step 1.1 --> delete unwanted & duplicates images [from 2 folders]. 

Armed with czkawka, I kicked off the mission. Apart from a minor hiccup, it was mostly smooth. The software is fantastic. Thanks, @‎qarmin & the community. Although it took me hours, it was pretty fast (excluding the scan time; after all, it was approx 300+ GB on HDD). I enjoyed the time and the revisit to our precious memories, especially with __almost__ no repetition.

I had meticulously selected the images to delete. Czkawka was configured to move the deleted files to trash.
I clicked `delete` and `Ok`
![Czkawka (Hiccup) 8_2_2022 18_21_13](https://user-images.githubusercontent.com/94835959/182465929-202a19d1-8b86-4586-b0d4-77e08c579d8a.png)
The next thing I saw was lots of errors stating - unable to delete so and so file.
I changed the configuration to delete files permanently. 
Clicked again on `delete` and nothing happened! To my horror, everything was deselected! And my time (almost a day)  wasted!

It would be awesome to have the selection exported to a file with the ability to import it later.
